app/controllers/verify/usps_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Verify
+  class UspsController < ApplicationController
+    include IdvStepConcern
+
+    before_action :confirm_step_needed
+
+    def index
+      @applicant = idv_session.resolution.vendor_resp.normalized_applicant
+    end
+
+    def create
+      idv_session.address_verification_mechanism = :usps
+      redirect_to verify_review_url
```

Comment:
`verify_review_path` ? I always use `_path` unless it's for a mailer or something external. any reason to use `url` instead?